content/en/platform/corda/5.0/application-networks/creating/members/cpi.md

```diff
@@ -9,6 +10,8 @@ menu:
 section_menu: corda5
 ---
 
+# Build and Upload the Member CPI
```

Comment:
Quick Q - can't you use some code `{{ .Title }}` instead of repeating the actual title ?

Reply:
Hopefully you can achieve same results - and then we avoid the "deja vu"

Reply:
I looked into that but unfortunately, you can't use variables in markdown files - only in shortcodes, etc.
I suppose you could create a shortcode that you call to add each H1 but didn't think there was much benefit to that over just writing the H1. I guess it would ensure the H1 and `title` always match 🤔

Reply:
Sorry - only read your second comment after posting my reply!

I think at this stage and with the planned move away from Hugo, it's probably not worth the extra work to create a shortcode and then redo all the H1s...